mne/gui/_coreg.py

```diff
@@ -215,10 +220,23 @@ def _set_current_fiducial(self, fid):
     def _set_info_file(self, fname):
         if fname is None:
             return
-        if not self._check_fif('info', fname):
+
+        # info file can be anything supported by read_raw
+        try:
+            check_fname(fname, 'info', tuple(raw_supported_types.keys()),
+                        endings_err=tuple(raw_supported_types.keys()))
+        except IOError as e:
+            warn(e)
+            self._widgets["info_file"].set_value(0, '')
             return
-        self._info_file = _check_fname(
-            fname, overwrite=True, must_exist=True, need_dir=False)
+
+        # ctf ds `files` are actually directories
+        if fname.endswith(('.ds',)):
+            self._info_file = _check_fname(
+                fname, overwrite=True, must_exist=True, need_dir=True)
```

Comment:
Are there any other datafile types that are directories and not files?

Reply:
Nirx I think (?), maybe others. It would probably be better to make _check_fname more flexible for example by using op.exists rather than isfile or isdir.

Reply:
Yes, nirx is a directory.

Reply:
`nirx` is supported via `read_raw` with the `.hdr` extension, so as a file. `coreg` users will need to pick the `hdr` file not just the directory for `nirx` datasets.

I did add a smirf (fixed a bug too) and nirx test though

@rob-luke, Its worth noting that I tried testing with
`snirf_nirsport2_20219_raw_path = op.join(data_path, 'SNIRF', 'NIRx', 'NIRSport2', '2021.9', '2021-10-01_002.snirf')` which threw an exception because the `Nasion` isn't defined in that file?

I agree about improving `_check_fname` but that would require an api change and be wide spread, so best left to another PR.

Reply:
Thanks for doing the additional testing @bloyl. It's good to know that the file doesn't contain nasion, that might be something we need to deal with in the future, or petition the vendor to export it if it's something we need.